src/components/Email.js

```diff
@@ -16,7 +16,7 @@ import "style/Email.scss";
 
 /* FORM */
 
-const validate = values => {
+export const validate = values => {
```

Comment:
another crazy idea: What do you think about collect validations that are shared by multiple components?